components/rule/org.wso2.carbon.application.deployer.rule/pom.xml

```diff
@@ -19,12 +19,12 @@
     <parent>
         <groupId>org.wso2.carbon.rules</groupId>
         <artifactId>rule</artifactId>
-        <version>4.4.10-SNAPSHOT</version>
+        <version>4.4.10</version>
```

Comment:
Do we need to manually change the version? Wouldn't Jenkins release build do that?

Reply:
fixed